lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -69,6 +69,7 @@
     "relative_humidity": Bounds((0, 1.2), "1"),
     "lwe_thickness_of_precipitation_amount": Bounds((0, 0.2), "m"),
     "lwe_thickness_of_precipitation_amount_in_vicinity": Bounds((0, 0.2), "m"),
+    "lwe_thickness_of_snowfall_amount": Bounds((0, 0.2), "m"),
```

Comment:
If I understand correctly, this is setting the highest likely quantity of new snow accumulating in (at most) a 24 hour period. This is currently set at 200mm lwe. The current record is 76 inches (1.93m) (Alaska, 1963). While this doesn't happen often, it probably isn't infrequent to get more than 200mm in mountains, so increasing this to 1 seems sensible. The same ought to be true for the precipitation amounts too. I'm not sure whether this would cause a problem for the shorter accumulation periods?

Reply:
I think it's a good idea to set higher bounds if needed; however are we sure the 1.93m is lwe?  If that's actual accumulated snowfall the rule-of-thumb is divide by 10,  which would be equivalent to 0.19m lwe - ie less than 0.2.  1 metre lwe snowfall would be 10 metres of lying snow accumulation...

Reply:
Oh yes, you're correct. The right LWE value is 0.193m, although probably a bit higher than this as that quantity of snow will compress under its own weight. I guess we can leave this value alone then.